mam/v2/api/api.c

```diff
@@ -23,20 +23,23 @@
  */
 
 static void mam_api_bundle_wrap(bundle_transactions_t *const bundle,
-                                trit_t *address, trits_t message) {
+                                trit_t const *const address,
+                                trit_t const *const tag, trits_t message) {
   iota_transaction_t transaction;
   flex_trit_t buffer[FLEX_TRIT_SIZE_6561];
   trits_t message_part = trits_null();
 
   transaction_reset(&transaction);
-  flex_trits_from_trits(buffer, NUM_TRITS_ADDRESS, address, NUM_TRITS_ADDRESS,
+  flex_trits_from_trits(buffer, NUM_TRITS_MESSAGE, address, NUM_TRITS_ADDRESS,
```

Comment:
this is not a guarantee, use `trits_size`

Reply:
Buffer is statically declared with a size of `FLEX_TRIT_SIZE_6561` which is the flex_trit equivalent of `NUM_TRITS_MESSAGE `